{
  "uid" : "881fcb81ca0f6970",
  "name" : "Add two numbers",
  "fullName" : "Add two numbers",
  "historyId" : "Add two numbers",
  "time" : {
    "start" : 1690625127711,
    "stop" : 1690625127714,
    "duration" : 3
  },
  "status" : "failed",
  "statusMessage" : "Connection refused (localhost:5001) -> Connection refused",
  "statusTrace" : "System.Net.Http.HttpRequestException: Connection refused (localhost:5001)\n ---> System.Net.Sockets.SocketException (111): Connection refused\n   at System.Net.Sockets.Socket.AwaitableSocketAsyncEventArgs.ThrowException(SocketError error, CancellationToken cancellationToken)\n   at System.Net.Sockets.Socket.AwaitableSocketAsyncEventArgs.System.Threading.Tasks.Sources.IValueTaskSource.GetResult(Int16 token)\n   at System.Net.Sockets.Socket.<ConnectAsync>g__WaitForConnectWithCancellation|281_0(AwaitableSocketAsyncEventArgs saea, ValueTask connectTask, CancellationToken cancellationToken)\n   at System.Net.Http.HttpConnectionPool.ConnectToTcpHostAsync(String host, Int32 port, HttpRequestMessage initialRequest, Boolean async, CancellationToken cancellationToken)\n   --- End of inner exception stack trace ---\n   at RestSharp.ResponseThrowExtension.ThrowIfError[T](RestResponse`1 response)\n   at RestSharp.RestClientExtensions.GetAsync[T](IRestClient client, RestRequest request, CancellationToken cancellationToken)\n   at LP.Tests.Specs.API.CalculatorApi.AddAsync() in /home/runner/work/testLP/testLP/src/LP.Tests/API/CalculatorApi.cs:line 29\n   at LP.Tests.Specs.Steps.CalculatorStepDefinitions.WhenTheTwoNumbersAreAddedAsync() in /home/runner/work/testLP/testLP/src/LP.Tests/Steps/CalculatorStepDefinitions.cs:line 35\n   at TechTalk.SpecFlow.Bindings.SynchronousBindingDelegateInvoker.<>c__DisplayClass2_0.<<InvokeBindingDelegateAsync>b__0>d.MoveNext()\n--- End of stack trace from previous location ---\n   at TechTalk.SpecFlow.Bindings.AsyncHelpers.<>c__DisplayClass1_0`1.<<RunSync>b__0>d.MoveNext()\n--- End of stack trace from previous location ---\n   at TechTalk.SpecFlow.Bindings.AsyncHelpers.ExclusiveSynchronizationContext.BeginMessageLoop()\n   at TechTalk.SpecFlow.Bindings.AsyncHelpers.RunSync[T](Func`1 task)\n   at TechTalk.SpecFlow.Bindings.SynchronousBindingDelegateInvoker.InvokeBindingDelegateAsync(Delegate bindingDelegate, Object[] invokeArgs)\n   at TechTalk.SpecFlow.Bindings.SynchronousBindingDelegateInvoker.InvokeDelegateSynchronously(Delegate bindingDelegate, Object[] invokeArgs)\n   at TechTalk.SpecFlow.Bindings.BindingInvoker.InvokeBinding(IBinding binding, IContextManager contextManager, Object[] arguments, ITestTracer testTracer, TimeSpan& duration)\n   at Allure.SpecFlowPlugin.AllureBindingInvoker.InvokeBinding(IBinding binding, IContextManager contextManager, Object[] arguments, ITestTracer testTracer, TimeSpan& duration)\n   at TechTalk.SpecFlow.Infrastructure.TestExecutionEngine.ExecuteStepMatch(BindingMatch match, Object[] arguments, TimeSpan& duration)\n   at TechTalk.SpecFlow.Infrastructure.TestExecutionEngine.ExecuteStep(IContextManager contextManager, StepInstance stepInstance)",
  "flaky" : false,
  "newFailed" : false,
  "beforeStages" : [ ],
  "testStage" : {
    "status" : "failed",
    "statusMessage" : "Connection refused (localhost:5001) -> Connection refused",
    "statusTrace" : "System.Net.Http.HttpRequestException: Connection refused (localhost:5001)\n ---> System.Net.Sockets.SocketException (111): Connection refused\n   at System.Net.Sockets.Socket.AwaitableSocketAsyncEventArgs.ThrowException(SocketError error, CancellationToken cancellationToken)\n   at System.Net.Sockets.Socket.AwaitableSocketAsyncEventArgs.System.Threading.Tasks.Sources.IValueTaskSource.GetResult(Int16 token)\n   at System.Net.Sockets.Socket.<ConnectAsync>g__WaitForConnectWithCancellation|281_0(AwaitableSocketAsyncEventArgs saea, ValueTask connectTask, CancellationToken cancellationToken)\n   at System.Net.Http.HttpConnectionPool.ConnectToTcpHostAsync(String host, Int32 port, HttpRequestMessage initialRequest, Boolean async, CancellationToken cancellationToken)\n   --- End of inner exception stack trace ---\n   at RestSharp.ResponseThrowExtension.ThrowIfError[T](RestResponse`1 response)\n   at RestSharp.RestClientExtensions.GetAsync[T](IRestClient client, RestRequest request, CancellationToken cancellationToken)\n   at LP.Tests.Specs.API.CalculatorApi.AddAsync() in /home/runner/work/testLP/testLP/src/LP.Tests/API/CalculatorApi.cs:line 29\n   at LP.Tests.Specs.Steps.CalculatorStepDefinitions.WhenTheTwoNumbersAreAddedAsync() in /home/runner/work/testLP/testLP/src/LP.Tests/Steps/CalculatorStepDefinitions.cs:line 35\n   at TechTalk.SpecFlow.Bindings.SynchronousBindingDelegateInvoker.<>c__DisplayClass2_0.<<InvokeBindingDelegateAsync>b__0>d.MoveNext()\n--- End of stack trace from previous location ---\n   at TechTalk.SpecFlow.Bindings.AsyncHelpers.<>c__DisplayClass1_0`1.<<RunSync>b__0>d.MoveNext()\n--- End of stack trace from previous location ---\n   at TechTalk.SpecFlow.Bindings.AsyncHelpers.ExclusiveSynchronizationContext.BeginMessageLoop()\n   at TechTalk.SpecFlow.Bindings.AsyncHelpers.RunSync[T](Func`1 task)\n   at TechTalk.SpecFlow.Bindings.SynchronousBindingDelegateInvoker.InvokeBindingDelegateAsync(Delegate bindingDelegate, Object[] invokeArgs)\n   at TechTalk.SpecFlow.Bindings.SynchronousBindingDelegateInvoker.InvokeDelegateSynchronously(Delegate bindingDelegate, Object[] invokeArgs)\n   at TechTalk.SpecFlow.Bindings.BindingInvoker.InvokeBinding(IBinding binding, IContextManager contextManager, Object[] arguments, ITestTracer testTracer, TimeSpan& duration)\n   at Allure.SpecFlowPlugin.AllureBindingInvoker.InvokeBinding(IBinding binding, IContextManager contextManager, Object[] arguments, ITestTracer testTracer, TimeSpan& duration)\n   at TechTalk.SpecFlow.Infrastructure.TestExecutionEngine.ExecuteStepMatch(BindingMatch match, Object[] arguments, TimeSpan& duration)\n   at TechTalk.SpecFlow.Infrastructure.TestExecutionEngine.ExecuteStep(IContextManager contextManager, StepInstance stepInstance)",
    "steps" : [ {
      "name" : "Given  the first number is 50",
      "time" : {
        "start" : 1690625127711,
        "stop" : 1690625127711,
        "duration" : 0
      },
      "status" : "passed",
      "steps" : [ ],
      "attachments" : [ ],
      "parameters" : [ ],
      "stepsCount" : 0,
      "attachmentsCount" : 0,
      "shouldDisplayMessage" : false,
      "hasContent" : false
    }, {
      "name" : "And  the second number is 70",
      "time" : {
        "start" : 1690625127711,
        "stop" : 1690625127711,
        "duration" : 0
      },
      "status" : "passed",
      "steps" : [ ],
      "attachments" : [ ],
      "parameters" : [ ],
      "stepsCount" : 0,
      "attachmentsCount" : 0,
      "shouldDisplayMessage" : false,
      "hasContent" : false
    }, {
      "name" : "When  the two numbers are added",
      "time" : {
        "start" : 1690625127711,
        "stop" : 1690625127713,
        "duration" : 2
      },
      "status" : "failed",
      "steps" : [ ],
      "attachments" : [ ],
      "parameters" : [ ],
      "stepsCount" : 0,
      "attachmentsCount" : 0,
      "shouldDisplayMessage" : false,
      "hasContent" : false
    }, {
      "name" : "Then  the result should be 120",
      "time" : {
        "start" : 1690625127713,
        "stop" : 1690625127713,
        "duration" : 0
      },
      "status" : "skipped",
      "steps" : [ ],
      "attachments" : [ ],
      "parameters" : [ ],
      "stepsCount" : 0,
      "attachmentsCount" : 0,
      "shouldDisplayMessage" : false,
      "hasContent" : false
    } ],
    "attachments" : [ ],
    "parameters" : [ ],
    "stepsCount" : 4,
    "attachmentsCount" : 0,
    "shouldDisplayMessage" : true,
    "hasContent" : true
  },
  "afterStages" : [ ],
  "labels" : [ {
    "name" : "thread",
    "value" : "NonParallelWorker"
  }, {
    "name" : "host",
    "value" : "fv-az339-242"
  }, {
    "name" : "feature",
    "value" : "Sample"
  }, {
    "name" : "tag",
    "value" : "mytag"
  }, {
    "name" : "resultFormat",
    "value" : "allure2"
  } ],
  "parameters" : [ ],
  "links" : [ ],
  "hidden" : false,
  "retry" : false,
  "extra" : {
    "severity" : "normal",
    "retries" : [ {
      "uid" : "1e3686de463e1c07",
      "status" : "failed",
      "statusDetails" : "Connection refused (localhost:5001) -> Connection refused",
      "time" : {
        "start" : 1690625127159,
        "stop" : 1690625127313,
        "duration" : 154
      }
    } ],
    "categories" : [ {
      "name" : "Product defects",
      "matchedStatuses" : [ ],
      "flaky" : false
    } ],
    "history" : {
      "statistic" : {
        "failed" : 6,
        "broken" : 0,
        "skipped" : 0,
        "passed" : 0,
        "unknown" : 0,
        "total" : 6
      },
      "items" : [ {
        "uid" : "116b862e27f94fc6",
        "reportUrl" : "https://apetrovskiy.github.io/testLP/12//#testresult/116b862e27f94fc6",
        "status" : "failed",
        "statusDetails" : "Connection refused (localhost:5001) -> Connection refused",
        "time" : {
          "start" : 1690623961398,
          "stop" : 1690623961401,
          "duration" : 3
        }
      }, {
        "uid" : "741ba3e56a4bc38b",
        "reportUrl" : "https://apetrovskiy.github.io/testLP/11//#testresult/741ba3e56a4bc38b",
        "status" : "failed",
        "statusDetails" : "Connection refused (localhost:5001) -> Connection refused",
        "time" : {
          "start" : 1690623930569,
          "stop" : 1690623930572,
          "duration" : 3
        }
      }, {
        "uid" : "99a3392c37aeebb6",
        "reportUrl" : "https://apetrovskiy.github.io/testLP/10//#testresult/99a3392c37aeebb6",
        "status" : "failed",
        "statusDetails" : "Connection refused (localhost:5001) -> Connection refused",
        "time" : {
          "start" : 1690620087575,
          "stop" : 1690620087578,
          "duration" : 3
        }
      }, {
        "uid" : "ff2c93b5d027e3ff",
        "reportUrl" : "https://apetrovskiy.github.io/testLP/9//#testresult/ff2c93b5d027e3ff",
        "status" : "failed",
        "statusDetails" : "Connection refused (localhost:5001) -> Connection refused",
        "time" : {
          "start" : 1690620060161,
          "stop" : 1690620060164,
          "duration" : 3
        }
      }, {
        "uid" : "480d6d0d6831fafb",
        "reportUrl" : "https://apetrovskiy.github.io/testLP/8//#testresult/480d6d0d6831fafb",
        "status" : "failed",
        "statusDetails" : "Connection refused (localhost:5001) -> Connection refused",
        "time" : {
          "start" : 1690619341771,
          "stop" : 1690619341775,
          "duration" : 4
        }
      } ]
    },
    "tags" : [ "mytag" ]
  },
  "source" : "881fcb81ca0f6970.json",
  "parameterValues" : [ ]
}